### PR TITLE
Add IFWI version struct

### PR DESCRIPTION
**Pull request by aravindksg**
Addresses: #213

### CODE REVIEW

**Comment by aravindksg** [comment]
@sanilkumar0 @jandres742 @joshuaranjan @saik-intel @wdamon-intel @jitendrasharma1989

File: scripts/sysman/firmware.yml [hunk 1]
```diff
@@ -31,6 +31,31 @@ members:
       name: "version[$S_STRING_PROPERTY_SIZE]"
       desc: "[out] NULL terminated string value. The string \"unknown\" will be returned if this property cannot be determined."
 --- #--------------------------------------------------------------------------
+type: struct
+desc: "Intel Firmware Image (IFWI) version"
```
Comment (jandres742) [inline scripts/sysman/firmware.yml:35]: could you make it experimental?

Comment (aravindksg) [reply scripts/sysman/firmware.yml]: @jandres742 moved to experimental extension

**Review by jandres742 — APPROVED** [review]

File: scripts/sysman/ifwiVersion.yml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+#
+# See YaML.md for syntax definition
+#
+--- #----------------------------------------------------------------------------------
+type: header
+desc: "Intel $OneApi Level-Zero Sysman Extension APIs for Intel Firmware Image Version"
+version: "1.8"
+--- #----------------------------------------------------------------------------------
+type: macro
+desc: "Intel Firmware Image Version Name"
```
Comment (wdamon-intel) [inline scripts/sysman/ifwiVersion.yml:14]: Why is this experimental extension vendor specific?

File: scripts/sysman/ifwiVersion.yml [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+version: "1.8"
+name: $S_FIRMWARE_IFWI_VERSION_EXP_NAME
+value: '"$XS_extension_ifwi_version"'
```
Comment (wdamon-intel) [inline scripts/sysman/ifwiVersion.yml:17]: Name should be `$XS_experimental_ifwi_version`

File: scripts/sysman/ifwiVersion.yml [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+etors:
+    - name: "1_0"
+      value: "$X_MAKE_VERSION( 1, 0 )"
+      desc: "version 1.0"
+--- #--------------------------------------------------------------------------
+type: struct
+desc: "Intel Firmware Image (IFWI) version"
+version: "1.8"
+class: $sFirmware
+name:
```
Comment (wdamon-intel) [inline scripts/sysman/ifwiVersion.yml:32]: Why is the name on a separate line?

**Review by wdamon-intel — CHANGES_REQUESTED** [review]

File: scripts/sysman/ifwiVersion.yml (hunk 2 above)
Comment (aravindksg) [reply scripts/sysman/ifwiVersion.yml:14]: yes, good point, shall follow up on this internally for any alternative solutions

**Comment by aravindksg** [comment]
Closing PR as we can't add vendor specific FW struct

**Pull request closed without merging**